modules/migration-manual-rollback.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+// migration/migrating-3-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+// migration/migrating-4-4/troubleshooting.adoc
+[id='migration-manual-rollback_{context}']
+= Manual rollback
+
+You can manually roll back a failed migration. Manual rollback is only required if an application has been quiesced, to prevent data corruption in the PV.
+
+This procedure is not required if the application was not quiesced, because the original application is still running on the source cluster.
+
+.Procedure
+
+. On the target cluster, switch to the migrated project:
++
+----
+$ oc project <project>
+----
+
+. Get the deployed resources:
++
+----
+$ oc get all
```

Comment:
Does this really get all resources now? There was a 3.x issue last week saying that it didn't.

Reply:
The command is being run on a 4.3 (or latest) cluster. 

Even if it doesn't actually get all the resources, that isn't critical. What's important is to kill anything that could be writing to the PV from the target cluster.

Reply:
Thanks for the clarification!